infra/repo_manager_test.py

```diff
@@ -118,7 +118,7 @@ def test_checkout_valid_pull_request(self):
       test_repo_manager = repo_manager.RepoManager(OSS_FUZZ_REPO, tmp_dir)
       test_repo_manager.checkout_pr('refs/pull/1757/merge')
       self.assertEqual(test_repo_manager.get_current_commit(),
-                       '2a2b11cc3d370db8f7bdf73046f3290a39615347')
+                       'e9c112b3b02debe90df1ef0395f5f6f64e3cd4db')
```

Comment:
Is it just me or do we frequently need to change these hashes in unrelated pull requests? 
Are the tests breaking frequently?

Reply:
I submitted a PR to oss-fuzz for testing purposes. #3415